docs/reference/contributing/guidelines/workflow.md

```diff
@@ -64,6 +64,8 @@ Pull requests on GitHub have to meet the following requirements to keep the code
 - Comment in the pull request on every change (rebase or new commits). This helps reviewers to be up to date with changes
 - Pull requests should fix a bug, add a feature or refactor.
 
+If commits do not follow above guidelines, you might be requested to rewrite the history (it is often adding more details to address _what_ and _why_ vs _how_).
```

Comment:
suggestion: requested to rewrite the history  -> requested to modify the commit history

Reply:
Changed